retrie.cabal

```diff
@@ -84,6 +84,7 @@ library
     directory >= 1.3.1 && < 1.4,
     filepath >= 1.4.2 && < 1.5,
     ghc >= 9.2,
+    ghc-boot,
```

Comment:
No constraints here, but that's partially because IIRC `ghc` itself tightly constraints what `ghc-boot` can be.

I can also import this from `template-haskell`, but that seems like a more onerous dependency.